### ISSUE TITLE
Compress font assets

### ISSUE THREAD

**Issue by gmarty** [aframevr/aframe#3891]
**Description:**

- A-Frame Version: 0.8.2
- Platform / Device: all
- Reproducible Code Snippet or URL: https://cdn.aframe.io/fonts/Roboto-msdf.png

I noticed that the PNG file for the Roboto font can be compressed:
<img width="635" alt="roboto" src="https://user-images.githubusercontent.com/333865/49476478-f1fccb80-f7ce-11e8-9435-cf71e1412d99.png">

Using ImageOptim with Insane optimisation level, I can squeeze 17% off the initial weight for the same quality.

I can push a branch with all the assets optimised, but in the long term, it'd be better to integrate it in the publishing workflow.

**Comment by dmarcos** [comment]
Thanks for looking at this. In principle I’m good with compressing. Any downsides we should be aware of?

**Comment by gmarty** [comment]
No, as long as you don't enable lossy compression, the resulting images are strictly identical to the input.

**Comment by ngokevin** [comment]
https://github.com/aframevr/assets/commit/6d8152cdd84e9a7690f020f2fae84c95a90a44c9

**Comment by ngokevin** [comment]
Did a quick compression, but reverted to test a bit more. Tried tinypng but realized it was lossy and might cause issues since it's fonts are very alpha-sensitive.
